### ISSUE TITLE
Add spec to test correct random distribution

### ISSUE THREAD

**Issue by mgreter** [sass/sass-spec#1450]
Not sure we have such a spec test yet, if so please ignore.

```
$lower: 0;
$upper: 0;

@for $i from 1 through 5000 {
  $random: random(2);
  @if ($random == 1) {
    $lower: $lower + 1;
  }
  @else {
    $upper: $upper + 1;
  }
}

result {
  lower: if($lower > 450, "OK", "Suspicious");
  upper: if($upper > 450, "OK", "Suspicious");
}

$lower: 0;
$upper: 0;

@for $i from 1 through 5000 {
  $random: random(null);
  @if ($random < 0.5) {
    $lower: $lower + 1;
  }
  @else {
    $upper: $upper + 1;
  }
}

result {
  lower: if($lower > 450, "OK", "Suspicious");
  upper: if($upper > 450, "OK", "Suspicious");
}
```

**Comment by nex3** [comment]
I'm not sure I'm a fan of adding a spec that only passes probabilistically. I feel like it's probably fairly easy to manually verify that a `random()` function actually produces different numbers in a fairly random way, and users aren't going to be using this for anything security-related so I don't think there's a pressing need to verify it with a high degree of certainty.

**Comment by mgreter** [comment]
I agree that this is not a test about anything security related, but I had made an error about `random(2)` that was not catched by any spec test, and I'm pretty sure we already have similar probabilistic test for random already.

**Comment by mgreter** [comment]
https://github.com/sass/sass-spec/blob/89531ae1c307bba140bf3d853599afa2ac55dc35/spec/core_functions/math/random.hrx

**Comment by nex3** [comment]
The existing `random()` tests are only probabilistic if the `random()` function is behaving incorrectly. If it's behaving correctly, they're guaranteed to pass.
